dallinger/prolific.py

```diff
@@ -281,3 +282,98 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
             raise ProlificServiceException(json.dumps(error))
 
         return parsed
+
+
+class DevProlificService(ProlificService):
+    """Wrapper that mocks the Prolific REST API and instead of making requests it writes to the log."""
+
+    def _req(self, method: str, endpoint: str, **kw) -> dict:
+        from uuid import uuid4
+
+        """Does NOT make any requests but instead writes to the log."""
+        self.debug_log_request(f"method = {method}, endpoint = {endpoint}, kw = {kw}")
```

Comment:
I think it would better to do 

```py
self.debug_log_request(method, endpoint, **kwarg)
```

and then leave the formatting to `debug_log_request`.

Reply:
Yes, did some more refactoring :+1: